### ISSUE TITLE
players are not able to connect

### ISSUE THREAD

**Issue by youmukonpaku1337** [KenRouKoro/C3H6N6O6#3]
https://filebin.net/gos2d17ukz3dhgmc
here's the log, would've used mclo.gs but the file doesnt fit into clipboard

**Comment by youmukonpaku1337** [comment]
note: players are using raknetify to connect but tcp doesn't work either

**Comment by youmukonpaku1337** [comment]
same thing happens with krypton or without it

**Comment by youmukonpaku1337** [comment]
cyclonite or the library mod for this is the cause

**Comment by youmukonpaku1337** [comment]
or from what it seems it's an incompatibility with packettweaker from polymer

**Comment by KenRouKoro** [comment]
Looking at the report it appears to be a problem with polymer translating network packets, I will try to fix this.

**Comment by youmukonpaku1337** [comment]
thanks!

**Comment by youmukonpaku1337** [comment]
oh by the way, please keep maintaining the 1.18.2 version, it's hard to update a large modded server

**Comment by youmukonpaku1337** [comment]
would be really nice to have this because the c2me+c3h6n6o6 duo would perform amazingly

**Comment by KenRouKoro** [comment]
It is like a truth in Chinese medicine: "Treat the symptoms but not the root of the disease. Treating the symptoms and the root cause is the only way to eradicate the disease.""治标不治本，病不除根。治标又治本，才能根除疾病。" C3H6N6O6 is the root of the system, while AC and lithium are the symptoms.

**Comment by terminalmaid** [comment]
don't really understand

**Comment by youmukonpaku1337** [comment]
so uh, any progress on polymer support?

**Comment by youmukonpaku1337** [comment]
hm

**Comment by youmukonpaku1337** [comment]
no progress at all?

**Comment by KenRouKoro** [comment]
I have to finish my final assignment first, I'm still a college student :D. I can't officially start the restoration until after I defend tomorrow afternoon.

**Comment by youmukonpaku1337** [comment]
ah i see

**Comment by KenRouKoro** [comment]
The latest version fixes a Mixin conflict with Polymer.

**Comment by youmukonpaku1337** [comment]
erm did you commit the changes?

**Comment by terminalmaid** [comment]
erm did you commit the changes?

**Comment by KenRouKoro** [comment]
Not yet, I'm still writing the logic for the client.

**Comment by youmukonpaku1337** [comment]
i see, also it's beta not bate

**Comment by KenRouKoro** [comment]
XD

**Comment by KenRouKoro** [comment]
Classic spelling mistakes

**Comment by youmukonpaku1337** [comment]
yeah

**Comment by youmukonpaku1337** [comment]
by the way, you might want to create a discord server for your mods

**Comment by youmukonpaku1337** [comment]
for faster support for people and suggestions

**Comment by KenRouKoro** [comment]
I will consider it

**Comment by youmukonpaku1337** [comment]
nice!

**Comment by KenRouKoro** [comment]
In fact both mcmtce and koroworld have discord servers, I just don't know how to use discord yet.

**Comment by KenRouKoro** [comment]
There is also the fact that discord does not work well in China and is often blocked by GFW.

**Comment by youmukonpaku1337** [comment]
right, you're in china
